cmd/server-rlimit.go

```diff
@@ -30,6 +34,9 @@ func setMaxResources() (err error) {
 		return err
 	}
 
+	// Set new max threads limit
+	increaseMaxThreads(int(maxLimit))
```

Comment:
> shouldn't we check if increaseMaxThreads returns maxLimit and return error if it doesn't?

Actually no. maxLimit could be less than the original value of MaxThreads. This function is not supposed to fail.